HomeEdu/Scenes/Storyboards/StoryBoards.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  StoryBoards.swift
+//  HomeEdu
+//
+//  Created by Nguyen Cuong on 7/30/19.
+//  Copyright © 2019 Nguyen Cuong. All rights reserved.
+//
+
+import UIKit
+enum Storyboards {
+    static let main = UIStoryboard(name: "Main", bundle: nil)
+    static let TabBar = UIStoryboard(name: "TabBar", bundle: nil)
```

Comment:
```suggestion
    static let tabBar = UIStoryboard(name: "TabBar", bundle: nil)
```
sai convention đặt tên biến, tất cả các property bên dưới cũng vậy